test/functional/test_framework/blocktools.py

```diff
@@ -150,10 +150,7 @@ def generate(node, n, preserve_utxos=[], send_witness=False):
         block = create_block(tip, coinbase, block_time)
         snapshot_meta = update_snapshot_with_tx(node, snapshot_meta, height + 1, coinbase)
         block.solve()
-        if send_witness:
```

Comment:
Now the variable `send_witness` is not used.

Reply:
Good point, didn't notice.